### PR TITLE
Add time sheet user name in invoice description

### PR DESCRIPTION
**Pull request by EBII**
in case of services the user who write time is not the client to invoice so it can read who made the job..

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash your commits, and send the CLA according https://odoo-community.org/page/cla

**Comment by EBII** [comment]
send the CLA according https://odoo-community.org/page/cla ... send it yesterday ;o) 
regards

File: sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py [hunk 1]
```diff
@@ -36,12 +38,110 @@ def test_sale_timesheet_description(self):
...
+        self.assertIn('Support Contract (on timesheet)\n'+str(date) + ' - ' +
+                      '10.5 Hour(s) - Test description 1234567890' +
+                      ' - Andrew Manager',
+                      description)
 
         self.default_timesheet_invoice_description = (
             self.SaleConfigSetting.create({}))
 
         self.default_timesheet_invoice_description.execute()
+
```
Comment (rousseldenis) [inline sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py:56]: @EBII Delete these comments or activate tests

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by EBII** [comment]
there is issue with this PR in case of module migration:
The choice get one more digits and all older settings became empty 
need to had 0 before in all value in base in case of upgrade.
maybe this year..

**Comment by mourad-ehm** [comment]
Code review :+1:

**Comment by pedrobaeza** [comment]
:-1: This is not valid while you:

* Uncomment tests and make them pass.
* Make migration script or don't change values.

**Pull request closed without merging**